lib/Agrammon/DB/Dataset.pm6

```diff
@@ -286,6 +296,25 @@ class Agrammon::DB::Dataset does Agrammon::DB {
         return self;
     }
 
+    method upload-data($content) {
+        my $fh = IO::String.new($content);
+        my $csv = Text::CSV.new;
+        my $i = 0;
+        while (my @row = $csv.getline($fh)) {
+            my ($var-name, $value) = @row;
+            next unless $var-name;
+            # skip comments
+            next if $var-name ~~ /^\#/;
+            self.store-input($var-name, $value);
+            $i++;
```

Comment:
Could be a `state` var

Reply:
What would be the benefit? I think I'd like to return the number of lines imported per upload, rather than the accumulated lines during a session (the usual use-case would be to upload an entire dataset).

Reply:
Ohh, I'd missed it was being returned. d'oh.